In [7]:
import pandas as pd

dados = pd.read_csv("/content/escritorio_energia.csv", encoding='latin1', sep=';')

media_consumo = dados["Consumo (kWh)"].mean()
reduzido = media_consumo * 0.75

economia_kwh = media_consumo - reduzido
co2_ev = economia_kwh * 0.116  # Fator médio de emissão CO2 por kWh no Brasil

print("=== Estimativa de Otimização ===")
print(f"Consumo médio atual: {media_consumo:.2f} kWh/dia")
print(f"Consumo otimizado estimado: {reduzido:.2f} kWh/dia")
print(f"Economia diária: {economia_kwh:.2f} kWh")
print(f"Redução mensal estimada de CO2: {co2_ev*30:.2f} kg")

=== Estimativa de Otimização ===
Consumo médio atual: 68.77 kWh/dia
Consumo otimizado estimado: 51.58 kWh/dia
Economia diária: 17.19 kWh
Redução mensal estimada de CO2: 59.83 kg


## Análise da Otimização de Consumo de Energia

Com base nos dados fornecidos, realizamos uma estimativa de otimização do consumo de energia no escritório. Os resultados são os seguintes:

*   **Consumo médio atual: 68.77 kWh/dia**
    *   Este valor representa o consumo diário médio de energia do escritório com base nos dados históricos. É o ponto de partida para a nossa análise.

*   **Consumo otimizado estimado: 51.58 kWh/dia**
    *   Este é o consumo diário projetado após a aplicação de medidas de otimização, que neste caso representam uma redução de 25% do consumo médio atual. Isso sugere que há um potencial significativo para diminuir o uso de energia.

*   **Economia diária: 17.19 kWh**
    *   A economia diária é a diferença entre o consumo médio atual e o consumo otimizado. Significa que o escritório poderia economizar 17.19 kWh por dia implementando as otimizações propostas.

*   **Redução mensal estimada de CO2: 59.83 kg**
    *   Esta métrica mostra o impacto ambiental da otimização. Considerando o fator médio de emissão de CO2 por kWh no Brasil (0.116 kg CO2/kWh), a economia de energia resultaria em uma redução de aproximadamente 59.83 kg de dióxido de carbono por mês. Isso demonstra um benefício ambiental significativo, contribuindo para a sustentabilidade.

Em resumo, a otimização proposta não apenas traria uma economia substancial no consumo de energia, mas também um impacto positivo na redução das emissões de gases de efeito estufa.